test/repository/SchedulesRepository.integration.spec.ts

```diff
@@ -72,11 +74,53 @@ describe('SchedulesRepository', () => {
       expect(schedules.filter((active) => active)).toHaveLength(1);
       expect(entities).toHaveLength(1);
     });
+
+    it('should replace dead schedules', async () => {
+      const active = await schedulesRepository.isActiveSchedule(scheduleName);
+      const secondScheduleId = 'not active';
+      const secondConnection = await Connection.create({ url: mongo.getUri() }, pingInterval, secondScheduleId);
+      const secondSchedulesRepository = await secondConnection.getSchedulesRepository();
+      const secondActive = await secondSchedulesRepository.isActiveSchedule(scheduleName);
+
+      expect(active).toEqual(true);
+      expect(secondActive).toEqual(false);
+
+      await sleep(1200);
+
+      const secondTakeOver = await secondSchedulesRepository.isActiveSchedule(scheduleName);
+      expect(secondTakeOver).toEqual(true);
+    });
+
+    it('should allow two active schedules with different names', async () => {
+      const active = await schedulesRepository.isActiveSchedule(scheduleName);
+      const otherScheduleName = 'other schedule';
```

Comment:
Either use only `other` or only `second` in variable names and strings. It's a bit confusing otherwise. :D